server/client_manager.py

```diff
@@ -905,6 +916,13 @@ def change_area(self, area, password=""):
                         f"[{self.id}] {self.showname} enters from [{old_area.id}] {old_area.name}{desc}",
                         "1",
                     )
+                    if self.area.area_manager.passing_msg == True:
+                        if showname == '':
+                            showname = self.char_name
+                        self.area.send_ic(
+                            None, '1', 1, "", "", f'~~[º{self.showname}º arrives.]', 
```

Comment:
This should mention from which area the client entered, similar to the OOC command.